autometrics-macros/.gitignore

```diff
@@ -0,0 +1 @@
+wip
```

Comment:
This is the artifact "work in progress" folder when working with trybuild. This is useful to ignore because there's always cruft there when making the test cases

Reply:
Ok, I wanted to avoid catching other files, but if it happens the gitignore can change again anyway